src/frontend/ts/types/urbitChess.ts

```diff
@@ -32,7 +32,9 @@ export enum Update {
   Position = 'position',
   Result = 'result',
   DrawOffer = 'draw-offer',
-  DrawDeclined = 'draw-declined'
+  DrawDeclined = 'draw-declined',
+  DrawClaimAvailable = 'draw-claim-available',
```

Comment:
`DrawClaimAvailable` isn't used, since the special draw status is sent as a part of the position update